server/src/test/java/org/apache/iotdb/db/metadata/MManagerBasicTest.java

```diff
@@ -1408,13 +1461,13 @@ public void testTotalSeriesNumber() throws Exception {
           CompressionType.GZIP,
           null);
       manager.createTimeseries(
-          new PartialPath("root.laptop.d1.s1.t1"),
+          new PartialPath("root.laptop.d1.s2.t1"),
           TSDataType.INT32,
```

Comment:
why does need alter test value ？

Reply:
The MeasurementMNode doesn't support addChild anymore, which is declared in jira 1495. The  related test case should be modified or deleted.